types/office-js-preview/index.d.ts

```diff
@@ -13056,11 +13056,12 @@ declare namespace Office {
          * @param options - An object literal that contains one or more of the following properties:-
          *        `asyncContext`: Developers can provide any object they wish to access in the callback function.
          * @param callback - Optional. When the method completes, the function passed in the `callback` parameter is called with a single parameter,
-         *                `asyncResult`, which is an `Office.AsyncResult` object.
+         *                `asyncResult`, which is an `Office.AsyncResult` object. The delivery date and time of a message is returned in the
+         *                `asyncResult.value` property. If a delivery date hasn't been set on a message yet, `0` is returned instead.
          *
          * @beta
          */
-        getAsync(options: Office.AsyncContextOptions, callback?: (asyncResult: Office.AsyncResult<Date>) => void): void;
+        getAsync(options: Office.AsyncContextOptions, callback?: (asyncResult: Office.AsyncResult<Date | 0>) => void): void;
```

Comment:
Can you verify this? I'm not sure returning 0 as the type of AsyncResult is correct - here and other instance
```suggestion
        getAsync(options: Office.AsyncContextOptions, callback?: (asyncResult: Office.AsyncResult<Date> | 0) => void): void;
```

Reply:
From the TypeScript Handbook, this would be classified as a numeric literal type (see [Literal Types](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-types)) since it would only ever return `0` if no delivery date has been set on the message.